tools/executors/translate/generate.ts

```diff
@@ -0,0 +1,34 @@
+import { readFileSync } from 'fs';
+import * as ts from 'typescript';
+
+export function generateStringLiteralsAndSourceFile(rootFilePath: string) {
+  const fileContent = readFileSync(rootFilePath, { encoding: 'utf-8' });
+
+  const sourceFile = ts.createSourceFile(
+    rootFilePath,
+    fileContent.toString(),
+    ts.ScriptTarget.ES2015,
+    true
+  );
+
+  const stringLiterals = getStringLiterals(sourceFile);
+  return { stringLiterals, sourceFile };
+}
+
+function getStringLiterals(node: ts.Node) {
+  let str: string[] = [];
+  node.forEachChild((childNode) => {
+    if (ts.isArrayLiteralExpression(childNode)) {
+      childNode.forEachChild((stringNode) => {
+        if (ts.isStringLiteral(stringNode)) {
```

Comment:
Can you please add examples in a comment that show which code you're trying to find in each line?

Btw, consider using ts-query.

Reply:
Done